examples/platform/esp32/common/CommonDeviceCallbacks.cpp

```diff
@@ -61,6 +62,43 @@ void CommonDeviceCallbacks::DeviceEventCallback(const ChipDeviceEvent * event, i
 
     case DeviceEventType::kCHIPoBLEConnectionClosed:
         ESP_LOGI(TAG, "CHIPoBLE disconnected");
+#if CONFIG_BT_ENABLED
+#if CONFIG_USE_BLE_ONLY_FOR_COMMISSIONING
+
+        if (chip::Server::GetInstance().GetFabricTable().FabricCount() > 0)
+        {
+            esp_err_t err = ESP_OK;
+#if CONFIG_BT_NIMBLE_ENABLED
+            if (ble_hs_is_enabled())
+            {
+                int ret = nimble_port_stop();
+                if (ret == 0)
+                {
+                    nimble_port_deinit();
+#if ESP_IDF_VERSION < ESP_IDF_VERSION_VAL(5, 0, 0)
+                    err = esp_nimble_hci_and_controller_deinit();
+#endif // ESP_IDF_VERSION < ESP_IDF_VERSION_VAL(5, 0, 0)
+#endif // CONFIG_BT_NIMBLE_ENABLED
+
+#if CONFIG_IDF_TARGET_ESP32
+                    err += esp_bt_mem_release(ESP_BT_MODE_BTDM);
+#elif CONFIG_IDF_TARGET_ESP32C2 || CONFIG_IDF_TARGET_ESP32C3 || CONFIG_IDF_TARGET_ESP32S3 || CONFIG_IDF_TARGET_ESP32H2
+            err += esp_bt_mem_release(ESP_BT_MODE_BLE);
```

Comment:
```suggestion
                    err += esp_bt_mem_release(ESP_BT_MODE_BLE);
```

Reply:
This was the change done by the restyler (locally). When I tried to fix it as above, the restyler check in CI fails. See this: https://restyled.io/gh/project-chip/repos/connectedhomeip/jobs/3099457/patch Don't know how :-D